source/faq.txt

```diff
@@ -26,8 +26,8 @@ progress.
 
 For best results, use MongoDB 3.2 or higher, which includes the
```

Comment:
See note below. It looks like MDB 3.6 is the earliest supported version for recent compass releases.